src/tlo/methods/symptommanager.py

```diff
@@ -601,6 +630,10 @@ def get_persons_with_newly_onset_symptoms(self):
     def reset_persons_with_newly_onset_symptoms(self):
         self._persons_with_newly_onset_symptoms.clear()
 
+    def get_current_symptoms(self, person_id: int) -> Set[str]:
```

Comment:
This looks good. 

So we should call this from "has what" for the special case of when it's a request for the symptoms for one person irrespective of disease module.